WordPress/Classes/Networking/PlansFeaturesRemote.swift

```diff
@@ -129,7 +129,6 @@ private func mapPlanFeaturesResponse(_ response: AnyObject) throws -> PlanFeatur
             let title = featureDetails["title"] as? String,
             var description = featureDetails["description"] as? String,
             let iconURLString = featureDetails["icon"] as? String,
```

Comment:
Could you also pull `iconURLString` out of the `if let`, so we don't fail if there's no url at all?